CONTRIBUTING.md

```diff
@@ -302,9 +175,17 @@ the documentation website.
 
 You can render the documentation locally to inspect the end result
 before creating a pull request. To compile the documentation locally:
-activate the `haddock3` python environment inside the haddock3 github
-folder, run `tox -e docs` and then use your favourite browser to open the
-`haddock3-docs/index.html` file. We invite you to read through Sphinx-doc
+- install HADDOCK in development mode with the `docs` extra requirements
+  (`pip install -e '.[docs]'`)
```

Comment:
```suggestion
- install HADDOCK in development mode with the `dev` and `docs` extra requirements
  (`pip install -e '.[dev,docs]'`)
```

Reply:
no need to install dev requirements to render the documentation